spec/models/review_spec.rb

```diff
@@ -44,4 +45,101 @@
   it "can create a review" do
     expect(Fabricate(:review)).to be_truthy
   end
+
+  it "can not create duplicate facility reviews" do
+    review = Fabricate(:review)
+    facility = Fabricate(:facility)
+    Fabricate(
+      :facility_review,
+      review: review,
+      facility: facility,
+      experience: :was_not_allowed
+    )
+    expect(review.facility_reviews.count).to eq(1)
+
+    # Generate a facility review for the same facility, should raise an error:
+    expect do
+      Fabricate(
+        :facility_review,
+        review: review,
+        facility: facility,
+        experience: :was_not_allowed
+      )
+    end.to raise_error(ActiveRecord::RecordInvalid)
+
+    expect(review.facility_reviews.count).to eq(1)
+  end
+end
+
+RSpec.describe Review, type: :request do
+  let(:user) { Fabricate(:user) }
+  let(:review) { Fabricate(:review, user: user) }
+  let(:space) { review.space }
+  let(:facility) { Fabricate(:facility) }
+  let(:facility_review) do
+    Fabricate(
+      :facility_review,
+      space: space,
+      user: user,
+      review: review,
+      facility: facility,
+      experience: :was_not_allowed
+    )
+  end
+
+  before do
+    sign_in user
+  end
+
+  it "can load the edit path" do
+    get edit_review_path(review)
+    expect(response).to have_http_status(:success)
+  end
+
+  it "has loaded the facility review, and set it to unlikely" do
+    facility_review.reload
+    expect(review.facility_reviews.count).to eq(1)
+    space.aggregate_facility_reviews
+    expect(space.reviews_for_facility(facility)).to eq("unlikely")
+  end
```

Comment:
Dette er en Model spec, ikke request spec.

Reply:
Den var mer ment som en test av om selve spec-en har loaded ting riktig. Om den testen der feiler, så kommer alle request specene til å feile, men fordi det ikke klarte laste ting. Kan kanskje bare fjerne den?

Reply:
Endret navn på den nå til `  it "has loaded the test data correctly for the request tests to come" `. Tror det er klarere. Har også tatt litt ut av den og opp til before, siden det må gjøres på alle.